### PR TITLE
Version 2.1.0 (Stable)

### PR DESCRIPTION
**Pull request by acro5piano**
I am really excited to release version 2.1.0, the first stable version of typed-graphqlify!

Thank you @luvies and all contributors! Your help have been greatly accelerating the development of this library!

I changed the API simpler, as #53 #51 #47 were merged.

Version 2.1.0 contains the following breaking changes:

**`query`, `mutation`, `subscription` are now top-level export**

Before:

```ts
import { graphqlify } from `typed-graphqlify`

graphqlify.query( //...
```

After:

```ts
import { query } from `typed-graphqlify`

query( //...
```

**`params` helper**

Before:

```ts
import { graphqlify, query } from `typed-graphqlify`

graphqlify.query({
  __params: { $input: 'UpdateUserInput' },
  updateUser: {
    __params: { input: $input },
   // ...
})
```

After:

```ts
import { params, query } from `typed-graphqlify`

query(params({ $input: 'UpdateUserInput' }, {
  updateUser: params({ input: $input }, {
   // ...
  }),
})
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,7 +87,9 @@ Note that we use our `types` helper to define types in the result, and the `para
 Then, convert the JS Object to GraphQL (string) with `graphqlify`:
 
 ```typescript
-const gqlString = graphqlify.query('getUser', getUserQuery)
+import { query } from 'typed-graphqlify'
```
Comment (acro5piano) [inline README.md]: I added these `import` statement to examples, because a lot of exports available now, and query examples are document in fact.

Comment (luvies) [reply README.md]: That's probably for the best, it's clearer to people looking over it quickly

**Comment by YardWill** [comment]
The API looks Great!

**Comment by acro5piano** [comment]
Thank you for your comments!

**Pull request merged**